models/src/main/java/com/vimeo/networking2/WatchedInteraction.kt

```diff
@@ -0,0 +1,30 @@
+package com.vimeo.networking2
+
+import java.util.*
+
+/**
+ * All actions for the watched list for a user.
+ */
+data class WatchedInteraction(
+
+    /**
+     * Whether the user has watched the video.
+     */
+    val added: Boolean? = null,
```

Comment:
I don't know if it's worth doing for such a small level of duplication, but `LikedInteraction`, `WatchLaterInteraction`, and `WatchedInteraction` all have the same exact properties. We could potentially create a single DTO like a `AddedInteraction` or something that contains all of these. Not sure it's worth it though since the amount of duplication is much smaller than the regular `Interaction` class.

Reply:
I will do this as part of the task for cleaning up the connections and interactions.